ddtrace/_trace/span.py

```diff
@@ -260,6 +256,13 @@ def duration(self, value):
         # type: (float) -> None
         self.duration_ns = int(value * 1e9)
 
+    @property
+    def sampled(self):
+        # type: () -> Optional[bool]
+        if self.context.sampling_priority is None:
+            return None
+        return self.context.sampling_priority > 0
```

Comment:
```suggestion
        return self.context.sampling_priority > 0
      
     @sampled.settr
     def sampled(self, value):
          @deprecated("pls don't do this, use set_tag(Manual keep/drop), this is a no-op"")
```

We should include a setter but make sure it logs a deprecation warning. This will maintain backwards compatibility. We should make sure we never use this internally